src/main/java/com/google/api/codegen/nodejs/NodeJSGapicContext.java

```diff
@@ -250,7 +250,7 @@ private String fieldParamComment(Field field, String paramComment, boolean isOpt
    */
   public List<String> fieldPropertyComment(Field field) {
     String commentType = fieldTypeCardinalityComment(field);
-    String fieldName = wrapIfKeywordOrBuiltIn(field.getSimpleName());
+    String fieldName = lowerUnderscoreToLowerCamel(field.getSimpleName());
```

Comment:
unnecessary -- actually existing `wrapIf...` isn't necessary, because this is a field type. In JS, something like `{if: 'foo'}` is syntactically valid.